lib/kitchen/directions/bake_chapter_references/v1.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeChapterReferences
+  class V1
+    renderable
+
+    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references')
+      @metadata = metadata_source.children_to_keep.copy
+      @klass = klass
+      @title = I18n.t(:references)
+      @uuid_prefix = uuid_prefix
+
+      chapter.references.search('h3').trash
+
+      chapter.non_introduction_pages.each do |page|
+        references = page.references
+        next if references.none?
+
+        references.search('h3').trash
+        title = Kitchen::Directions::EocSectionTitleLinkSnippet.v1(page: page)
+
+        references.each do |reference|
+          reference.prepend(child: title)
+        end
+      end
+
+      references = chapter.pages.references.cut
```

Comment:
If you use this in the template, I think it could be just one like, put an “@“ to references and paste it in the template

Reply:
Done!

Reply:
Since this uses the common eoc template, I have to go by it's rules, and it takes a string, not a clipboard

Reply:
oh right, approved!